addons/account/models/account_move_line.py

```diff
@@ -38,6 +38,7 @@ class AccountMoveLine(models.Model):
         ondelete="cascade",
         check_company=True,
     )
+    suitable_journal_ids = fields.Many2many(related='move_id.suitable_journal_ids')
```

Comment:
I don't know why this would be needed, this field is used to build the domain of `account.move`, and we never set a domain on the journal of `account.move.line`?
Unless the domain is inherited for related fields? :thinking: 
Then it means that we need to have a related for all the fields that are part of the domain of a related?

I would almost prefer setting the domain explicitly to `[]` then, since the field is readonly anyways (and really impossible to change from the line itself)

Reply:
the domain is inherited for related fields

Reply:
With @rco-odoo  we change the fields.py to fix inconsistencies. The domain would only be used if it is not a string or it is an inherited field.